File: dart_programs/practise_programs/reverse_words.dart


void main() {
  String str = "My name is Vikas";
  List<String> words = str.split(" ");
  String result = "";
  for (String word in words) {
    String revWord = "";
    for (int i = word.length - 1; i >= 0; i--) {
      revWord += word[i];
    }
    result += revWord + " ";
  }
  print(result.trim());
}

File: striver_series/src/ReverseWords.java

public class ReverseWords {

    public static void main(String[] args) {
        String str = "My name is Vikas";
        String[] words = str.split("\\s+");
        String result = "";
        for( String word : words){
            String revWord = "";
            for(int i = word.length()-1;i>=0;i--){
                revWord += word.charAt(i);
            }
            result += revWord +" ";
        }
        System.out.println(result);
    }
}